dev/doc/critical-bugs

```diff
@@ -167,6 +167,19 @@ Universes
   GH issue number: none
   risk: unlikely to be triggered in interactive mode, not present in batch mode (i.e. coqc)
 
+  component: algebraic universes
+  summary: Set+2 was incorrectly simplified to Set+1
+  introduced: V8.10 (with the SProp commit 75508769762372043387c67a9abe94e8f940e80a)
+  impacted released: V8.10.0 V8.10.1 V8.10.2
```

Comment:
It's supposed to be "impacted released versions" which could be shortened to "impacted releases" but not "impacted released". The same error appears in the entry just above, from which you probably copied the template.

```suggestion
  impacted released versions: V8.10.0 V8.10.1 V8.10.2
```

Reply:
@SkySkimmer you still need to fix this

Reply:
What's there to fix?

Reply:
Took care of it finally, this will be faster.